src/Resources/contao/classes/Backend.php

```diff
@@ -886,9 +886,11 @@ public static function addPagesBreadcrumb($strKey='tl_page_node')
 		// Insert breadcrumb menu
 		$GLOBALS['TL_DCA']['tl_page']['list']['sorting']['breadcrumb'] .= '
 
-<ul id="tl_breadcrumb">
-  <li>' . implode(' &gt; </li><li>', $arrLinks) . '</li>
-</ul>';
+<nav aria-label="' . $GLOBALS['TL_LANG']['MSC']['breadcrumbAriaLabel'] . '" role="navigation">
```

Comment:
If there are several navigation elements on one page, it is very helpful for screen reader users if you label them differently so that the user is able to distinguish their usage. Something like "main navigation", "breadcrumb navigation" etc.